index.html

```diff
@@ -187,5 +187,21 @@ <h3 class="conn">LET’S CONNECT</h3>
           </div>
 
     </section>
+ 
+    <section class="contact_section">
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-content">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!
+        </p>
+        <form class="contact-form" action="https://formspree.io/f/xeqwngzr" method="POST">
+            <input type="text" name="name" placeholder="Lorem Ipsum" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" placeholder="loremipsum@gmail.com" placeholder="Email" required>
+            <textarea  name="message" maxlength="500" required>
+                Write your message here...
+            </textarea>
+
+            <button class="contact-btn" type="submit">Get in touch</button>
+        </form>
+    </section>
```

Comment:
- [ ] [REQUIRED] The margin top between the contact section and the preceeding section is much more than you have in your design at the moment. Please ensure you stick to the design template details as much as possible. 

Figma design | Your work
---- | ----
![image](https://user-images.githubusercontent.com/84156714/235708303-b73e2aec-2b15-4de7-bac1-847b10142b94.png) | ![image](https://user-images.githubusercontent.com/84156714/235708489-c796cd6d-7c5f-42fd-ba57-1c2d9bc2dd28.png)